controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/ApplicationSerializer.java

```diff
@@ -124,11 +119,6 @@
 
     // JobStatus field
     private static final String jobTypeField = "jobType";
-    private static final String errorField = "jobError";
```

Comment:
Does the previous version handle removal of these fields?

Reply:
Yes, all fields were already optional, except the job type.